.github/buildomat/jobs/build-and-test-linux.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+#:
+#: name = "build-and-test (ubuntu-20.04)"
+#: variety = "basic"
+#: target = "ubuntu-20.04"
```

Comment:
Matrix-like support would be nice for Buildomat in the future, since these scripts are otherwise mostly the same.

Reply:
I agree!  I added a few cross-platform shims to the images at the time so that they could be identical without effectively an ifdef soup: https://github.com/oxidecomputer/buildomat#available-commands

Matrix support is definitely on my list.